api-report/runtime-utils.api.md

```diff
@@ -72,6 +73,26 @@ export function exceptionToResponse(err: any): IResponse;
 // @public (undocumented)
 export type Factory = IFluidDataStoreFactory & Partial<IProvideFluidDataStoreRegistry>;
 
+// @internal
+export class GCDataBuilder implements IGarbageCollectionData {
```

Comment:
This looks like a graph algorithm thing.  I think it sounds appealing to group / name everything below that layer of abstraction together.  i.e. anything that deals only in id -> outbound routes graph structure.

Reply:
That's a good point. We can move them together, may be in a separate PR?